tests/Auth/AuthorizeMiddlewareTest.php

```diff
@@ -86,6 +86,42 @@ public function testSimpleAbilityAuthorized()
         $this->assertEquals($response->content(), 'success');
     }
 
+    public function testSimpleAbilityWithStringParameter()
+    {
+        $this->gate()->define('view-dashboard', function ($user, $param) {
+            return $param === 'true';
+        });
+
+        $this->router->get('dashboard', [
+            'middleware' => Authorize::class.':view-dashboard,true',
+            'uses' => function () {
+                return 'success';
+            },
+        ]);
+
+        $response = $this->router->dispatch(Request::create('dashboard', 'GET'));
+
+        $this->assertEquals($response->content(), 'success');
```

Comment:
I guess this is actually the other way round ;-)
`$this->assertEquals('success', $response->content());`

Reply:
Happy to change this, every other assertion like this in the file is `$this->assertEquals($response->content(), 'success')` though.